EncounterHandler.js

```diff
@@ -689,12 +689,10 @@ function inject_monster_image(stat) {
 		return;
 	}
 	let url = stat.data.largeAvatarUrl;
-	if (url === undefined || url == null || url.length == 0) {
-		url = stat.data.avatarUrl;
-	}
+
 	if (url !== undefined && url != null && url.length > 0 && window.EncounterHandler.combat_body.find(".encounter-details-content-section__content .injected-image").length == 0) {
-		let content = window.EncounterHandler.combat_body.find(".encounter-details-content-section__content");				
-		let image = `<img style="width:100%" class="injected-image" src="${url}" alt="${stat.data.name}" class="monster-image">`;
+		let content = window.EncounterHandler.combat_body.find(".encounter-details-content-section__content");
+		let image = `<img style="width:100%" class="injected-image" src="${url}" alt="${stat.data.name}" class="monster-image" onerror="this.src='${stat.data.avatarUrl}'"></img>`;
```

Comment:
The last thing I think you need here is to add `; onerror=''` so it ends up like this

    onerror="this.src='${stat.data.avatarUrl}';onerror='';"

Otherwise we could end up in an infinite loop if `stat.data.avatarUrl` also errors out